nvflare/lighter/impl/auth_policy.py

```diff
@@ -20,6 +20,18 @@
 
 class AuthPolicyBuilder(Builder):
     def __init__(self, orgs, roles, groups, disabled):
+        """Creates and writes authorization.json to the server's startup directory with the authorization policy
+        defining the groups each org is in and the admin client roles which controls the allowed rights. The
+        participant information from project.yml is included in authorization.json with what orgs, groups, and roles
+        are associated with each participant. This builder also checks for errors if the arguments are specified
+        incorrectly.
+
+        Args:
+            orgs: authorization configuration for orgs (it may be helpful to build this section with the UI)
+            roles: authorization configuration for roles (it may be helpful to build this section with the UI)
+            groups: authorization configuration for groups (it may be helpful to build this section with the UI)
+            disabled: if true, all users are super with all privileges
```

Comment:
All looks good.

Just one question, what thing does this argument "disabled" disable?

Reply:
If disabled, this builder just enters the 'researcher' mode and makes everyone with role = super.  That is, it disables the authorization portion so everyone has the highest rights.